ubuntu-headless-qt/README.md

```diff
@@ -0,0 +1,996 @@
+# RZG2L SBC board #
+This is the quick startup guide for RZG2L SBC board (hereinafter referred to as `RZG2L-SBC`).
+The below will describe the current status of development, how to build, set up environment for RZG2L-SBC.
+
+## Status
+This is a Ubuntu release of the RZG2L development product for RZG2L-SBC.
+
+This release provides the following features:
+
+ - Ubuntu build compatible with RZG2L SoC
+ - RZG2L-SBC Linux BSP functionalities
+ - 40 IO expansion interface supported
+ - On-board Wireless Modules enabled (only support for Wi-Fi)
+ - On-board Audio Codec with Stereo Jack Analog Audio IO
+ - Generic USB Bluetooth framework supported
+ - MIPI DSI enabled
+ - Bootloader with U-Boot Fastboot UDP enabled.
+
+Known issues:
+
+ - Only support for 48 Khz audio sampling rate family.
+
+## Building Ubuntu
```

Comment:
Are these steps for building Yocto? Why are you putting them here?

If your expectation is just let users know that the "core-image-qt-rzpi.tar.bz2" is required for the build. Just mention that.

Reply:
These steps are for running main_script.sh, I will remove some redundant guides (such as install package and setup git configuration)

Reply:
I have replaced all step below to the new one, referring Ubuntu System Porting Guide for Renesas RZ/G2L-based Remi Pi